modules/claims_api/spec/factories/auto_established_claims.rb

```diff
@@ -106,4 +106,66 @@
     termnl_digit_nbr { Faker::Number.number(digits: 2) }
     filed5103_waiver_ind { 'Y' }
   end
+  factory :bgs_response_without_lc_status, class: OpenStruct do
+    benefit_claim_details_dto { (association :bgs_claim_details_dto_without_lc_status).to_h }
```

Comment:
@FonzMP BGS response. Take note of the detail label: `benefit_claim_details_dto`